cba_settings.sqf

```diff
@@ -1,13 +1,13 @@
 // A3TI
-force force A3TI_ACE_JAVELIN = false;
-force force A3TI_ALLOW_TANK_DRIVER = false;
-force force A3TI_ALLOW_VANILLA_TI = true;
-force force A3TI_ENABLE_ZEUS = true;
-force force A3TI_HD_VISION_FIGHTER = false;
-force force A3TI_HD_VISION_LANDVEH = true;
-force force A3TI_HD_VISION_SHIP = false;
-force force A3TI_HD_VISION_UAV = true;
-force force A3TI_REMOVE_FILMGRAIN_RHS = true;
+A3TI_ACE_JAVELIN = false;
+A3TI_ALLOW_TANK_DRIVER = false;
+A3TI_ALLOW_VANILLA_TI = true;
+A3TI_ENABLE_ZEUS = true;
+A3TI_HD_VISION_FIGHTER = false;
+A3TI_HD_VISION_LANDVEH = true;
+A3TI_HD_VISION_SHIP = false;
+A3TI_HD_VISION_UAV = true;
+A3TI_REMOVE_FILMGRAIN_RHS = true;
```

Comment:
This allow for player desired settings else cScripts will make this settings for you and you are forced to change it to yours each mission..
```suggestion
//A3TI_ACE_JAVELIN = false;
//A3TI_ALLOW_TANK_DRIVER = false;
//A3TI_ALLOW_VANILLA_TI = true;
//A3TI_ENABLE_ZEUS = true;
//A3TI_HD_VISION_FIGHTER = false;
//A3TI_HD_VISION_LANDVEH = true;
//A3TI_HD_VISION_SHIP = false;
//A3TI_HD_VISION_UAV = true;
//A3TI_REMOVE_FILMGRAIN_RHS = true;
```